otter/test/rest/test_groups.py

```diff
@@ -152,8 +166,30 @@ def test_format_state_different_status(self, config_value):
             True,
             ScalingGroupStatus.ERROR,
             desired=10)
-        result = format_state_dict(state, active)
+        result = self.format(state, active)
         self.assertEqual(result['status'], 'ERROR')
+        self.assertEqual(result['errors'], [])
```

Comment:
That is true but that condition is not something which this function can ensure. I could change the test with a reason but I not sure if it buys us something apart from avoiding confusion?